### ISSUE TITLE
[BUG]: New version X.XX.X is not signed by the application owner: publisherNames:

### ISSUE THREAD

**Issue by AndresMorelos** [AndresMorelos/Invoncify#62]
**Describe the bug**

**To Reproduce**
1. Install any previous version of Invoncify on a windows machine.
2. Check for updates
3. Download the update

**Expected behavior**
The app needs to auto-update by itself

**Screenshots**
```
Error: New version 1.13.2 is not signed by the application owner: publisherNames: Developer ID Application: Andres Camilo Morelos Negrette (7U3PPW87MJ), raw info: {
  "SignerCertificate": {
    "FriendlyName": "",
    "IssuerName": {
      "Name": "C=US, O=...
```

**Desktop (please complete the following information):**
 - OS: Windows

**Comment by AndresMorelos** [comment]
A Code Sing certificate is needed to put off this issue: https://codesigningstore.com/microsoft-authenticode-signing?gclid=CjwKCAjwve2TBhByEiwAaktM1HKc0DzB2lAvy9h1JX4Vd7mo3loYKy0X64AaL1UsbWj-Gsim9pRp3xoClSEQAvD_BwE

**Comment by mcfalld** [comment]
getting the same one for 1.21, should I just download the new one separately?

**Comment by AndresMorelos** [comment]
@mcfalld Yes, for window users you will need to download the installer again or you can use the winget CLI.

 I need to purchase a Code Sing Certificate to enable the auto-update again.

Don't worry your data will not be lost.
